code/modules/surgery/organs/organ.dm

```diff
@@ -32,14 +32,6 @@ var/list/organ_cache = list()
 	var/sterile = 0 //can the organ be infected by germs?
 	var/tough = 0 //can organ be easily damaged?
 
-/obj/item/organ/Destroy()
```

Comment:
Um, this isn't the one I said to remove; the one I saw only had 

```
/obj/item/organ/Destroy()
	if(owner)
		remove(owner, 1)
	return ..()
```
in it. This one is fine. Where's the other one?